bundle.Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23 as builder
+WORKDIR /go/src/github.com/openshift/lws-operator
+COPY . .
+
+ARG OPERAND_IMAGE=registry.stage.redhat.io/lws-operator/lws-rhel9@sha256:c6c825ba5bf39ab1e2e38190bef3f3d40eef3702ffb372ccf973e31aec043e80
+ARG REPLACED_OPERAND_IMG=\${OPERAND_IMAGE}
```

Comment:
`${OPERAND_IMAGE}` is an image placeholder which will be brought by https://github.com/openshift/lws-operator/pull/15. This `${OPERAND_IMAGE}` (and `${OPERATOR_IMAGE}`) will be replaced by Konflux with the latest image.